google/cloud/internal/backoff_policy.cc

```diff
@@ -26,7 +26,7 @@ std::unique_ptr<BackoffPolicy> ExponentialBackoffPolicy::clone() const {
 
 std::chrono::milliseconds ExponentialBackoffPolicy::OnCompletion() {
   using std::chrono::duration_cast;
-  using std::chrono::microseconds;
+  using microseconds = std::chrono::duration<double, std::micro>;
```

Comment:
I would proffer that this is more confusing than it's worth ... having `microseconds` and `milliseconds` be different types of things is just confusing.

[In my diff I added a `DoubleMicroseconds` private alias for the former.]

Nit: After that, I'd probably just expand the few uses of the other names, getting rid of the `using`s altogether.

Reply:
Changed the using to DoubleMicroseconds and removed the other two